Source/Csla/IContainsDeletedList.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
```

Comment:
This needs the header comment block like other files.

Reply:
I always struggle with wording/naming.  Hopefully that is clear.